src/components/MerchantPayout/MerchantPayoutScreen.tsx

```diff
@@ -0,0 +1,153 @@
+
+const styles = StyleSheet.create({
+  content: {
+    position: "relative",
+    padding: size(2),
+    paddingVertical: size(8),
+    height: "100%",
+    width: 512,
+    maxWidth: "100%"
+  },
+  headerText: {
+    marginBottom: size(4)
+  },
+  bannerWrapper: {
+    marginBottom: size(1.5)
+  },
+  buttonsWrapper: {
+    marginTop: size(2),
+    flexDirection: "row",
+    alignItems: "center"
+  },
+  submitWrapper: {
+    flex: 1,
+    marginRight: size(1)
+  }
+});
+
+export const MerchantPayoutScreen: FunctionComponent<NavigationFocusInjectedProps> = ({
+  navigation
+}) => {
+  const messageContent = useContext(ImportantMessageContentContext);
+  const { config } = useConfigContext();
+  const showHelpModal = useContext(HelpModalContext);
+  const [vouchers, setVouchers] = useState([{ valid: 5, invalid: 3 }]);
+  const [merchantCode, setMerchantCode] = useState("");
+
+  const submitMerchantCode = (): void => {
+    if (!validateMerchantCode(merchantCode)) {
+      return Alert.alert(
+        "Error",
+        "Invalid Merchant Code",
+        [
+          {
+            text: "Dismiss"
+          }
+        ],
+        {}
+      );
+    }
+    Alert.alert("Valid Merchant Code");
+  };
+
+  const onCancel = (): void => {
+    Alert.alert("Warning", "Cancelling clears all vouchers. Are you sure?", [
+      {
+        text: "Yes",
+        onPress: () => {
+          setVouchers([]);
+          console.log("Yes Pressed");
+        }
+      },
+      {
+        text: "No",
+        onPress: () => console.log("No Pressed")
+      }
+    ]);
+  };
+
+  return (
+    <>
+      <ScrollView
+        contentContainerStyle={{ alignItems: "center" }}
+        scrollIndicatorInsets={{ right: 1 }}
+        keyboardShouldPersistTaps="handled"
+      >
+        <KeyboardAvoidingView behavior="position">
+          <TopBackground mode={config.appMode} />
+          <View style={styles.content}>
+            <View style={styles.headerText}>
+              <AppHeader mode={config.appMode} />
+            </View>
+            {messageContent && (
+              <View style={styles.bannerWrapper}>
+                <Banner {...messageContent} />
+              </View>
+            )}
+            <Card>
+              <VoucherInputSection
+                vouchers={vouchers}
+                merchantCode={merchantCode}
+                setMerchantCode={setMerchantCode}
+                submitMerchantCode={submitMerchantCode}
+              />
+            </Card>
+            <View style={styles.buttonsWrapper}>
+              <View style={styles.submitWrapper}>
+                <DarkButton
+                  fullWidth={true}
+                  text="Checkout"
+                  icon={
+                    <Feather
+                      name="shopping-cart"
+                      size={size(2)}
+                      color={color("grey", 0)}
+                    />
+                  }
+                  onPress={submitMerchantCode}
```

Comment:
Perhaps can name this better, since checking out does not only submit the merchant code, but also the vouchers that have been redeemed

Reply:
Alright, we renamed it to 'redeemVouchers'!